partner_nace/wizard/nace_import.py

```diff
@@ -0,0 +1,282 @@
+# -*- coding: utf-8 -*-
+# Python source code encoding : https://www.python.org/dev/peps/pep-0263/
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright :
+#        (c) 2015 Antiun Ingenieria, SL (Madrid, Spain, http://www.antiun.com)
+#                 Antonio Espinosa <antonioea@antiun.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, api, _
+from openerp.exceptions import Warning
+import requests
+import re
+import logging
+from lxml import etree
+from collections import OrderedDict
+
+logger = logging.getLogger(__name__)
+
+
+class NaceImport(models.TransientModel):
+    _name = 'nace.import'
+    _description = 'Import NACE activities from European RAMON service'
+    _parents = [False, False, False, False]
+    _available_langs = {
```

Comment:
Because there is not a direct relation between Odoo language codes and NACE languages. If you notice, NACE language code sometimes refers to language and sometimes to country. For example, Norwegian Bokmål or Slovenian